strawberry/cli/commands/server.py

```diff
@@ -15,8 +14,16 @@
 @click.argument("module", type=str)
 @click.option("-h", "--host", default="0.0.0.0", type=str)
 @click.option("-p", "--port", default=8000, type=int)
-def server(module, host, port):
-    sys.path.append(os.getcwd())
+@click.option(
+    "--app-dir",
```

Comment:
@BryceBeagle what do you think of `--app-dir` vs `--src-dir`?

Reply:
I think `--src-dir` is more reflective of what the option actually does, but consistency is nice. I don't have a strong opinion on which is used.

Regardless, I think the `help` argument to the `@click.option` should provide a little note mentioning that this is the same functionality as the `--app-dir` flag that the uvicorn CLI provides.

Reply:
@bryanculver The help message is copied almost verbatim from the uvicorn CLI. Are you suggesting to make it refer to the uvicorn CLI for context? It's already kinda verbose.

<img width="619" alt="Screenshot 2021-04-05 at 18 24 15" src="https://user-images.githubusercontent.com/880829/113597649-4f71e300-963c-11eb-9f0c-0e4a9078b018.png">

Reply:
I assume you meant to tag me instead :stuck_out_tongue:. I was just suggesting adding a quick blurb that says something like "This is the same functionality as the `--app-dir` option that Uvicorn provides". I don't think it would be too verbose :slightly_smiling_face: